### PR TITLE
Created login page

### LINKED ISSUE #20: Feature request: Sign up page
**User Story**
For example: 
As a user
I want to be able to create an account
So I can log in the app

**Priority**
[*]: High
[]: Medium
[]: Low

**Describe the solution you want to implement**
There is a page for creating an account where a user can submit their username and password so they are able to log in the app in the future.

**Describe alternatives you've considered**
No account set up page, an account is created automatically if the system does not recognize the credentials.

**Issue Dependencies**
Issue #13 
**Additional context**
n/a

### LINKED ISSUE #44: Feature Request: Log in page
**User Story**
As a user, 
I want to be able to log in the app,
So I can use it

**Priority**
[X]: High
[]: Medium
[]: Low

**Describe the solution you want to implement**
A screen based on the design mockup where a user can enter their credentials and log in the app. The user is then taken to their transaction list (#32).

**Describe alternatives you've considered**
n/a

**Issue Dependencies**
n/a

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by babe269**
Closes #44 

**Description**

Added the basic screen based design of the mock up login page. Integrated with the routing of other pages within the application. Had assistance from @tgre605, @williamlixu  and @misl000  

Split logo was added to the pages folder.Login page was modified with the new content.
Template for Sign up page has been added along with assistance from @tgre605 to be completed.

**Testing**

Ran and functional.

**Checklist**

- [x] I have tested my change
- [x] Code builds successfully and all tests pass
- [x] Docs have been added/updated if needed

### CODE REVIEW

**Review by tgre605 — APPROVED** [review]
Looks good

**Review by tgre605 — DISMISSED** [review]
Looks good to me

File: split-webapp/split/src/pages/Login.js [hunk 1]
```diff
@@ -1,22 +1,99 @@
...
+import {NavLink} from  "react-router-dom";
+import mainLogo from'./split2.png';
+
+
+
+
+
+
+const useStyles = makeStyles(theme => ({
+    outterContainer: {
```
Comment (williamlixu) [inline split-webapp/split/src/pages/Login.js]: This should be moved to the CSS file

**Review by williamlixu — CHANGES_REQUESTED** [review]
LGTM, just small fix to move CSS files. Also the existing CSS is in PascalCase, so would be good to match it :)

**Review by tgre605 — APPROVED** [review]
Looks good

File: split-webapp/split/src/App.css [hunk 2]
```diff
@@ -157,3 +157,44 @@ display: flex;
...
+  margin-top: 2.5em;
+}
+
+.SignIn {
+  font-size: 10%;
+  color: white;
+  margin-left: 5%;
+}
+
+/* .TextField {
```
Comment (williamlixu) [inline split-webapp/split/src/App.css]: Clean up this CSS please :)

**Review by williamlixu — CHANGES_REQUESTED** [review]
Looks like there's a merge conflict in the CSS file now

**Review by williamlixu — APPROVED** [review]
LGTM

**Comment by williamlixu** [comment]
Also closes #40, but logo still needs to be added elsewhere (only on home/login pages currently).

**Pull request merged**